src/index.ts

```diff
@@ -2,8 +2,9 @@ import { Api } from './eosjs-api';
 import * as ApiInterfaces from './eosjs-api-interfaces';
 import { JsonRpc } from './eosjs-jsonrpc';
 import * as Numeric from './eosjs-numeric';
+import * as KeyConversions from './eosjs-key-conversions';
 import * as RpcInterfaces from './eosjs-rpc-interfaces';
 import { RpcError } from './eosjs-rpcerror';
 import * as Serialize from './eosjs-serialize';
 
-export { Api, ApiInterfaces, JsonRpc, Numeric, RpcInterfaces, RpcError, Serialize };
+export { Api, ApiInterfaces, JsonRpc, Numeric, KeyConversions, RpcInterfaces, RpcError, Serialize };
```

Comment:
Things which depend on ec shouldn't be in this bundle

Reply:
Removed